src/main/java/Flight.groovy

```diff
@@ -0,0 +1,15 @@
+import groovy.transform.ToString
+
+import java.time.LocalDateTime
+
+@ToString(includeNames = true, includeFields = true)
+class Flight {
+    String id
```

Comment:
I think you are building the ID out from the other fields - From, destination and a number? Maybe id can be a method that calculates it . just a suggestions